docs/schema/manageddatatype-element-sps15xsdsearchset2.md

```diff
@@ -1,26 +1,25 @@
 ---
 title: "ManagedDataType element (SPS15XSDSearchSet2)"
+description: Outlines information, a definition, and elements and attributes for the ManagedDataType element in Sharepoint.
 manager: arnek
-ms.date: 3/9/2015
+ms.date: 06/09/2022
 ms.audience: ITPro
 ms.topic: article
 ms.prod: sharepoint
 ms.localizationpriority: medium
 ms.assetid: a7ed1865-1281-a5ee-485d-40ae30db6555
-description: "Last modified: March 09, 2015"
 ---
 
 # ManagedDataType element (SPS15XSDSearchSet2)
 
  
   
 ## Element information
+**Element type**: tns:ManagedDataType
```

Comment:
We've been directed to keep Markdown tables as tables. You can use "Information/Location" or "Name/Value" as headers, or if you find a more specific header that fits, use that.